tests/dataset_builders/pie/test_cdcp.py

```diff
@@ -0,0 +1,431 @@
+import dataclasses
+from typing import List
+
+import pytest
+from datasets import disable_caching, load_dataset
+from pytorch_ie import DatasetDict, tokenize_document
+from pytorch_ie.annotations import LabeledSpan
+from pytorch_ie.core import AnnotationList, Document, annotation_field
+from pytorch_ie.documents import (
+    TextBasedDocument,
+    TextDocumentWithLabeledSpansAndBinaryRelations,
+)
+from transformers import AutoTokenizer, PreTrainedTokenizer
+
+from dataset_builders.pie.cdcp.cdcp import (
+    CDCP,
+    CDCPDocument,
+    convert_to_text_document_with_labeled_spans_and_binary_relations,
+    document_to_example,
+    example_to_document,
+)
+from src.document.types import TokenDocumentWithLabeledSpansAndBinaryRelations
+from tests import FIXTURES_ROOT
+from tests.dataset_builders.common import _deep_compare
+
+disable_caching()
+
+DATASET_NAME = "cdcp"
+SPLIT_SIZES = {"train": 581, "test": 150}
+# HF_DATASET_PATH = "DFKI-SLT/cdcp"
+HF_DATASET_PATH = CDCP.BASE_DATASET_PATH
+PIE_DATASET_PATH = "pie/cdcp"

```

Comment:
This should point to the local script in the `dataset_builders` folder:
```suggestion
PIE_DATASET_PATH = PIE_BASE_PATH / DATASET_NAME
```

Using `pie/cdcp` would result in [the script hosted at the Huggingface hub] (https://huggingface.co/datasets/pie/cdcp) (because it can be found under that path). But we want to test what's in this repo.

Reply:
fix in ec6b346